src/native/corehost/hostmisc/pal.unix.cpp

```diff
@@ -424,35 +417,68 @@ bool pal::get_dotnet_self_registered_dir(pal::string_t* recv)
     FILE* install_location_file = pal::file_open(install_location_file_path, "r");
     if (install_location_file == nullptr)
     {
-        trace::verbose(_X("The install_location file failed to open."));
+        trace::error(_X("The install_location file failed to open."));
```

Comment:
Please also print out error details - at least errno and what file path we were trying to open.
(Recently debugged a case in Azure where it was hard to get the verbose tracing, but errors are readily available).